Makefile

```diff
@@ -17,7 +17,11 @@ VERSION=$(shell $(PYTHON) setup.py --version 2>/dev/null)
 DESTDIR=/
 AVOCADO_DIRNAME=$(shell echo $${PWD\#\#*/})
 AVOCADO_EXTERNAL_PLUGINS=$(filter-out ../$(AVOCADO_DIRNAME), $(shell find ../ -maxdepth 1 -mindepth 1 -type d))
-AVOCADO_OPTIONAL_PLUGINS=$(shell find ./optional_plugins -maxdepth 1 -mindepth 1 -type d)
+# List of optional plugins is now static, because there may be depedencies between plugins
+AVOCADO_OPTIONAL_PLUGINS="./optional_plugins/html"
+AVOCADO_OPTIONAL_PLUGINS+="./optional_plugins/runner_remote"
+AVOCADO_OPTIONAL_PLUGINS+="./optional_plugins/runner_vm"
+AVOCADO_OPTIONAL_PLUGINS+="./optional_plugins/runner_docker"
```

Comment:
I see, how about something like this:

```
AVOCADO_OPTIONAL_PLUGINS="./optional_plugins/runner_remote"
AVOCADO_OPTIONAL_PLUGINS+="./optional_plugins/runner_vm"
AVOCADO_OPTIONAL_PLUGINS+="./optional_plugins/runner_docker"
AVOCADO_OPTIONAL_PLUGINS+=`find ./optional_plugins -maxdepth 1 -mindepth 1 -type d`
AVOCADO_OPTIONAL_PLUGINS=$AVOCADO_OPTIONAL_PLUGINS | uniq
```
(rewritten to Makefile but basically the idea is to first hardcode the ones which depends on something, then add all of them and then use uniq to avoid duplicates)